src/components/Alert/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import Alert from '.';
+
+describe('Alert', () => {
+  const errors = ['Error 1'];
+  const dataTestId = 'test-id__alert';
```

Comment:
You can name the `data-test-id` with BEM. Having the rule to follow can help us, and other projects are using the same 😄 
```suggestion
  const dataTestId = 'alert';
```